doc/nrf/app_dev/bootloaders_dfu/sysbuild_image_ids.rst

```diff
@@ -0,0 +1,88 @@
+| ``SB_CONFIG_MCUBOOT_MAX_UPDATEABLE_IMAGES``            | Maximum number of MCUboot images                                                                                                                     |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES``                | Number of MCUboot images to set in images                                                                                                            |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``SB_CONFIG_MCUBOOT_ADDITIONAL_UPDATEABLE_IMAGES``     | Number of additional MCUboot images to include in MCUboot (described below)                                                                          |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``SB_CONFIG_MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES``       | If enabled then will set both MCUboot and the main application's number of MCUboot images, if not set then will only set this is the MCUboot image   |
++--------------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+MCUboot-only images
+*******************
+
+There are images that are only visible to MCUboot, this allows for MCUboot to deal with those images without interference/knowledge from the application, or to allow for features like shared secondary slots.
+At present, this is used by MCUboot to update MCUboot itself, which can enabled with secure boot, this will share the main application's secondary slot for storing firmware updates but will move the update directly to the opposing MCUboot image slot which the xNSIB?x can then boot (the application does not need to be aware of how this works, just that it can place an MCUboot update as it would a normal application update and reboot to apply).
+To allow for this, there are additional MCUboot-only images which are not presented to the main application and are only set in MCUboot.
+The total number of images set in MCUboot is ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES + SB_CONFIG_MCUBOOT_ADDITIONAL_UPDATEABLE_IMAGES``, whilst in the main application it is just ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES`` (if ``SB_CONFIG_MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES`` is enabled, otherwise will not be set)
```

Comment:
```suggestion
Some images are visible only to MCUboot, allowing it to manage them independently of the main application.
This setup supports features like shared secondary slots. 
Currently, MCUboot uses this capability to update itself, which can be enabled with secure boot. 
MCUboot then shares the main application's secondary slot for firmware updates.
However, it directly transfers the update to the designated MCUboot image slot, which is then booted by |NSIB|.

The main application does not need to be aware of this process; it only needs to handle an MCUboot update as it would any normal application update and reboot to apply changes.
This functionality is possible because there are additional MCUboot-only images that are not accessible to the main application and are configured only within MCUboot. 

The total number of images configured in MCUboot is the sum of ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES`` and ``SB_CONFIG_MCUBOOT_ADDITIONAL_UPDATEABLE_IMAGES``.
If ``SB_CONFIG_MCUBOOT_APP_SYNC_UPDATEABLE_IMAGES`` is enabled, the main application sets only ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES``. 
Otherwise, it is not set.
```

Reply:
Fixed